dronegen/gha.go

```diff
@@ -16,92 +16,132 @@ package main
 
 import (
 	"fmt"
+	"path"
 	"sort"
 	"strings"
 	"time"
 
 	"golang.org/x/exp/maps"
 )
 
-type ghaBuildType struct {
-	buildType
-	trigger
-	pipelineName      string
-	ghaWorkflow       string
+type ghaWorkflow struct {
+	name              string
+	stepName          string
 	srcRefVar         string
-	workflowRef       string
+	ref               string
 	timeout           time.Duration
 	slackOnError      bool
-	dependsOn         []string
 	shouldTagWorkflow bool
 	seriesRun         bool
 	inputs            map[string]string
 }
 
-func ghaBuildPipeline(b ghaBuildType) pipeline {
-	p := newKubePipeline(b.pipelineName)
-	p.Trigger = b.trigger
+type ghaBuildType struct {
+	buildType
+	trigger
+	pipelineName string
+	checkoutPath string
+	dependsOn    []string
+	workflows    []ghaWorkflow
+	// ghaWorkflow       string
+	// srcRefVar         string
+	// workflowRef       string
+	// timeout           time.Duration
+	// slackOnError      bool
+
+	// shouldTagWorkflow bool
+	// seriesRun         bool
+	// inputs            map[string]string
```

Comment:
This comment can probably be removed, now that it is in the ghaWorkflow struct

Reply:
Ah yep thank you - forgot to delete